### DISCUSSION TITLE
servo engine for Lazarus/free pascal

### DISCUSSION THREAD

**Discussion by Reaper10** [servo/servo#28524] [Ideas]
I think it be great for the servo engine be made for Lazarus IDE/free pascal. That way someone using Lazarus IDE/free pascal could make something using the servo engine.
https://www.freepascal.org/
https://www.lazarus-ide.org/

**Comment by jdm** [comment]
This should be possible through the libsimpleservo C embedding layer: https://github.com/servo/servo/tree/master/ports/libsimpleservo/capi
